lms/templates/instructor/edx_ace/allowedenroll/email/body.html

```diff
@@ -0,0 +1,62 @@
+{% extends 'ace_common/edx_ace/common/base_body.html' %}
+
+{% load i18n %}
+{% load static %}
+{% block content %}
+<table width="100%" align="left" border="0" cellpadding="0" cellspacing="0" role="presentation">
+    <tr>
+        <td>
+            <h1>
+                {% blocktrans %}You have been invited to {{ course_name }}{% endblocktrans %}
+            </h1>
+
+            <p style="color: rgba(0,0,0,.75);">
+                {% blocktrans %}You have been invited to join {{ course_name }} at {{ site_name }} by a member of this course staff.{% endblocktrans %}
+                <br />
+            </p>
+
+        {% if is_shib_course %}
+            {% if auto_enroll %}
+                <p style="color: rgba(0,0,0,.75);">
+                    {% blocktrans %}To access this course click on the button below and login:{% endblocktrans %}
+                    <br />
+                </p>
+
+                {% include "ace_common/edx_ace/common/return_to_course_cta.html" with course_cta_text=course_name course_cta_url=course_url %}
+            {% elif course_about_url is not None %}
+                <p style="color: rgba(0,0,0,.75);">
+                    {% blocktrans %}To access this course visit this course and register:{% endblocktrans %}
+                    <br />
+                </p>
+
+                {% include "ace_common/edx_ace/common/return_to_course_cta.html" with course_cta_text=course_name course_cta_url=course_about_url %}
+            {% endif %}
+        {% else %}
+            <p style="color: rgba(0,0,0,.75);">
+                {% blocktrans %}Please finish your registration and fill out the registration form making sure to use {{ email_address }} in the Email field:{% endblocktrans %}
+                <br />
+            </p>
+
+            {% trans "Finish Your Regsitration" as button_cta_text %}
```

Comment:
Typo: Registration

Reply:
Thanks @nasthagiri! Good catch :slightly_smiling_face: 

Fixed it.